chsdi/templates/htmlpopup/geschiebemessnetz.mako

```diff
@@ -0,0 +1,164 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<% c['stable_id'] = True %>
+<%
+    lang = 'fr' if lang in ('fr', 'it') else 'de'
+%>
```

Comment:
For the table below, we have `or '-'` clauses, here not. We should be consistent for both, no?